README.md

```diff
@@ -72,32 +121,49 @@ func sayHelloToScratchy() {
+
+Let's try to generalize `sayHelloToGarfield()`, etc., a bit, so they consist of just _one_ function that uses an argument. Here's how you would define a more general function, `sayHelloToCat()`:
 
 ```swift
-func sayHelloToCat(kittenName: String) {
-    print("Hello, \(kittenName), why do you sleep so much?")
+func sayHelloToCat(catName: String) {
+    print("Hello, \(catName), why do you sleep so much?")
 }
+```
 
-func sayHelloTDog(puppyName: String) {
-    print("Hello, \(puppyName), want to go sniff that rock together?")
-}
+Enter the above code into your playground file. Take a look at what you did: You defined a function (using the `func` keyword) called `sayHelloToCat`. Like `sayHello`, you opened with a set of parentheses. But instead of being empty, you declared an argument called `catName` of type `String`. Then, you wrote a function body consisting of a call to `print()`. Notice that the string you are printing interpolates the argument, `catName`. `catName` is a constant that you can use anywhere inside the `sayHelloToCat()` function. You can think of it as being a constant declared inside of the function body, just like `greeting` was declared inside of `sayHello()`. Like `greeting`, `catName` is visible only within the _scope_ of `sayHelloToCat`, so you cannot refer to it outside of the function.
+
+You can call `sayHelloToCat()` just like you called `sayHello()`, except you have to pass an argument to `sayHelloToCat()`. You can simply pass a string:
 
+```swift
+sayHelloToCat("Mittens")
 ```
 
+Now you should see "Hello, Mittens, why do you sleep so much?" printed to the playground's console.
 
-* So we created a function, but now how do we call on it?
-* These functions take in String types as their arguments, which is something we know how to do create.
-* Show them these two ways of passing in Strings to this function:
+You can also declare a constant or variable, then pass the constant or variable to `sayHelloToCat()`.
 
 ```swift
-let nameOfCatEatingLasagna = "Garfield"
-sayHelloToCat(nameOfCatEatingLasagna)
-// prints "Hello, Garfield, why do you sleep so much?"
+let catName = "Socks"
+sayHelloToCat(catName)
+```
+
+(Remember: That constant or variable _must_ be a `String`, because you declared the argument to `sayHelloToCat()` to be a `String`!)
+
+Because `sayHelloToCat()` takes an argument, you cannot call it without one. Try calling it without an argument in your playground file and see what happens:
 
-sayHelloToCat("Max")
-// prints "Hello, Max, why do you sleep so much?"
+```swift
+sayHelloToCat()  // You'll get an error here!
 ```
 
+Do you know why Swift couldn't run that line of code?
+
```

Comment:
As simple as this might be for us to answer, the student (without seeing an answer to this particular question) might think too deep about it. I think it's cool to provide them with the answer to this question after posing the question. For future reference though, I don't think it's ALWAYS required to provide them with the answer.